### DISCUSSION TITLE
[Python] How to show warnings for undefined (non-defined) variables?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [VSCodium/vscodium#1350] [Q&A]
I installed VSCodium. I installed Python extension. Python server is "default".

In any file, even if I remove all imports, there are no warnings about undefined local variables. Auto-completion works.

Is this a problem with VSCodium, or with "Jedi", or with "Jedi python-language-server"?
Or, maybe, I need to add some custom settings?

**Comment by fellanonymous** [comment]
I've ran into this problem too. Have you found a solution @simon-liebehenschel ? Anyone else know what's going on?
